tools/stack_decode.py

```diff
@@ -29,8 +29,8 @@ def decode_stacktrace_log(input_source):
   # bazel-out/local-dbg/bin/source/server/_virtual_includes/backtrace_lib/server/backtrace.h:84]
   backtrace_marker = "\[backtrace\] [^\s]+"
   trace_begin_re = re.compile(
-      "^(.+)%s Backtrace obj<(.+)> thr<(\d+)" % backtrace_marker)
-  stackaddr_re = re.compile("%s thr<(\d+)> #\d+ (0x[0-9a-fA-F]+)$" % backtrace_marker)
+      "^(.+)%s Backtrace thr<(\d+).*obj<(.+)>" % backtrace_marker)
```

Comment:
Any particular reason for using `"thr<(\d+).*obj<(.+)>"` instead of `"thr<(\d+)> obj<(.+)>"`? We know exactly the format that's emitted.